assets/javascripts/kitten/hoc/media-queries.test.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { mediaQueries } from 'kitten/hoc/media-queries'
+
+const SimpleComponent = ({
+  viewportIsMobile,
+  viewportIsTabletOrLess,
+  ...props
+}) => {
+  return (
+    <div title="Test me!" { ...props } />
+  )
+}
+
+describe('mediaQueries()', () => {
+  const originalMatchMedia = window.matchMedia
+  let SimpleComponentWithMediaQueries
+  let component
+
+  beforeEach(() => {
+    window.matchMedia = jest.fn()
+  })
+
+  afterEach(() => {
+    window.matchMedia = originalMatchMedia
+  })
+
+  describe('by default', () => {
```

Comment:
Je pense qu'il faudrait ajouter un test pour vérifier que les listeners sont bien mis ou que les callbacks sont déclenchées lorsqu'on match une media query.

Reply:
C'est noté 👍